iOS/DuckDuckGo/SaveCreditCard/SaveCreditCardViewController.swift

```diff
@@ -19,11 +19,11 @@
 
 import UIKit
 import BrowserServicesKit
+import Core
 import SwiftUI
 
 protocol SaveCreditCardViewControllerDelegate: AnyObject {
     func saveCreditCardViewController(_ viewController: SaveCreditCardViewController, didSaveCreditCard card: SecureVaultModels.CreditCard)
-    func saveCreditCardViewControllerDidCancel(_ viewController: SaveCreditCardViewController)
```

Comment:
I’ve removed this completely since it was only dismissing the sheet in TabViewController (which is now being handled here)